src/ops.jl

```diff
@@ -182,3 +182,28 @@ end
 function (*){FSV <: FixedVector}(a::FSV, b::FSV)
     FixedMatrix{eltype(FSV), 1, 1}(dot(a,b))
 end
+
+function (==){T1,T2,C,N}(a::FixedArray{T1,C,N}, b::FixedArray{T2,C,N})
+    l = prod(N.parameters)
```

Comment:
No reason, but `length` is more understandable. Either way, it should become a constant.